docs/installation-deb.md

```diff
@@ -81,9 +53,39 @@ Supported distributions:
 * Debian 7 (wheezy), 8 (jessie)
 * Centos 6, 7
 
+### Install Metrictank
+You can enable our repository and install the metrictank package like so:
+
+```
+curl -s https://packagecloud.io/install/repositories/raintank/raintank/script.deb.sh | bash
+apt-get install metrictank
+```
 [more info](https://packagecloud.io/raintank/raintank/install)
 
 
+### Install Graphite
+
+Install Graphite via your prefered method as detailed at http://graphite.readthedocs.io/en/latest/install.html
+(We hope to provide Debian and Ubuntu packages in the near future.)
+
+Configure graphite with the following settings in local_settings.py
+```
+CLUSTER_SERVERS = ['localhost:6060'] # update to match the host:port metrictank is running on.
```

Comment:
this is still a big improvement on the previous instructions, which didnt even mention configuring graphite-api.

refactoring this install guide is outside the scope of this PR.